### PR TITLE
Convert to runtime: ruby.

### PR DESCRIPTION
**Pull request by jeffmendoza**

### CODE REVIEW

File: app.yaml [hunk 1]
```diff
@@ -12,10 +12,11 @@
 # limitations under the License.
 
 # [START runtime]
-runtime: custom
+runtime: ruby
 vm: true
 api_version: 1
 module: default
+entrypoint: bundle exec rackup -p 8080 -E production /app/config.ru
```
Comment (JustinBeckwith) [inline app.yaml]: @dazuma should we still need the /app/config.ru here?  Or just config.ru?

File: app.yaml [hunk 2]
```diff
@@ -12,10 +12,11 @@
 # limitations under the License.
 
 # [START runtime]
-runtime: custom
+runtime: ruby
 vm: true
 api_version: 1
```
Comment (JustinBeckwith) [inline app.yaml]: dump api_version

Comment (jeffmendoza) [inline app.yaml]: will do

File: app.yaml (hunk 1 above)
Comment (dazuma) [inline app.yaml]: Just config.ru

**Comment by JustinBeckwith** [comment]
LGTM

**Pull request merged**